pkg/kvs/kvs.go

```diff
@@ -1,34 +1,74 @@
 package kvs
 
 import (
-	"errors"
+	"context"
+	"encoding/hex"
 
-	"github.com/ethersphere/bee/pkg/api"
-	"github.com/ethersphere/bee/pkg/kvs/manifest"
-	"github.com/ethersphere/bee/pkg/kvs/memory"
+	"github.com/ethersphere/bee/pkg/file"
+	"github.com/ethersphere/bee/pkg/manifest"
 	"github.com/ethersphere/bee/pkg/swarm"
 )
 
-var ErrInvalidKvsType = errors.New("kvs: invalid type")
-
 type KeyValueStore interface {
-	Get(rootHash swarm.Address, key []byte) ([]byte, error)
-	Put(rootHash swarm.Address, key, value []byte) (swarm.Address, error)
+	Get(key []byte) ([]byte, error)
+	Put(key, value []byte) error
+	Load() manifest.Interface
```

Comment:
We should be able to change the kvs implementation to pot, but we can't if there is manifest related types on the interface

Reply:
This is a good point. If there is no easy solution for now, can we put a TODO comment on Load() to make it more generic later?

Reply:
Load has been removed. As of now it does not have any purpose.